canonicalwebteam/discourse_docs/app.py

```diff
@@ -59,7 +59,7 @@ def document_view(path=""):
                 except RedirectFoundError as redirect:
                     return flask.redirect(redirect.target_url)
                 except PathNotFoundError:
-                    return flask.abort(404)
+                    return flask.redirect("/")
```

Comment:
@nottrobin In this case do we keep the 404?

Reply:
Please leave this as a 404.

Reply:
:+1: